index.js

```diff
@@ -20,7 +20,7 @@ function isFunction(f) {
 
 module.exports = {
 
-  browser: process.browser,
+  browser: typeof process === "undefined" || !process.nextTick || Boolean(process.browser),
```

Comment:
I think `process` is supposed to be undefined in the browser -- open up a Chrome console and type `process`.  Is it defined?